amlb/datasets/openml.py

```diff
@@ -34,40 +34,27 @@ def __init__(self, api_key, cache_dir=None):
         if cache_dir:
             oml.config.set_cache_directory(cache_dir)
 
+        if oml.config.retry_policy != "robot":
+            log.debug("Setting openml retry_policy from '%s' to 'robot'." % oml.config.retry_policy)
+            oml.config.set_retry_policy("robot")
```

Comment:
just wondering if a similar logic is not needed in `amlb/benchmarks/openml.py`

Reply:
I never experienced the issues myself (I don't have the permissions to spawn instances in those numbers). Though that's partially the reason I had it in `runbenchmark.py`. I'll just add it there as well to ensure robot policy is used in all stages.